pkg/dhclient/dhclient.go

```diff
@@ -126,6 +146,22 @@ func lease4(ctx context.Context, iface netlink.Link, timeout time.Duration, retr
 }
 
 func lease6(ctx context.Context, iface netlink.Link, timeout time.Duration, retries int) (Lease, error) {
+	// For ipv6, we cannot bind to the port until Duplicate Address
+	// Detection (DAD) is complete which is indicated by the link being no
+	// longer marked as "tentative". This usually takes about a second.
+	for {
+		if ready, err := isIpv6LinkReady(iface); err != nil {
+			return nil, err
+		} else if ready {
+			break
+		}
+		select {
+		case <-time.After(100 * time.Millisecond):
```

Comment:
did you mean for this to have a `fallthrough`?

Reply:
Added continue to make it more clear.